angular/pxb-component-theme.scss

```diff
@@ -94,13 +94,17 @@
             background-color: map-get($background, background);
         }
 
-        .pxb-info-list-item-active .pxb-info-list-item .mat-list-item-content {
+        .pxb-info-list-item-active * {
```

Comment:
Do we really want `*` here? Every single item in the list-item-active should be color=primary[500]?

Reply:
I think this is fine; the only thing that concerned me was that the status color might be affected by this, but it isn't due to the color being an inline-style.

![image](https://user-images.githubusercontent.com/6538289/91593165-a9375f80-e92d-11ea-852d-20259623bcb9.png)